resources/ts/Components/IntegrationTypes.tsx

```diff
@@ -75,10 +90,23 @@ export type IntegrationTypesInfo = ReturnType<
 export const IntegrationTypes = () => {
   const integrationTypesInfo = useIntegrationTypesInfo();
 
+  const filteredIntegrationTypes = uitpasEnabled
```

Comment:
Maybe make a comment and a ticket, so we don't forget to remove this code once the feature toggle is not necessary anymore...

Reply:
There is already a ticket to remove the feature toggle, see https://jira.publiq.be/browse/PPF-481

I added a note to also remove this const `filteredIntegrationTypes`